libminifi/src/FlowControlProtocol.cpp

```diff
@@ -280,11 +280,11 @@ int FlowControlProtocol::sendReportReq() {
   data = this->encode(data, hdr.msgType);
   data = this->encode(data, hdr.seqNumber);
   data = this->encode(data, hdr.status);
-  data = this->encode(data, hdr.payloadLen);
+  this->encode(data, hdr.payloadLen);
```

Comment:
I think this should not have been changed: now `FLOW_YML_NAME` starts writing the buffer at the same point as where `hdr.payloadLen` started, overwriting it.